src/Mammon/Program.cs

```diff
@@ -89,7 +90,7 @@
     var subActor = ActorProxy.Create<ISubscriptionActor>(new ActorId("uniphar-dev"), "SubscriptionActor",
         new ActorProxyOptions { RequestTimeout = Timeout.InfiniteTimeSpan });
 
-//await subActor.RunWorkload(new Mammon.Models.Actors.CostReportRequest { SubscriptionName = "uniphar-dev", CostFrom = DateTime.UtcNow.AddDays(-31), CostTo = DateTime.UtcNow.AddDays(-1) });
+    await subActor.RunWorkload(new CostReportRequest { SubscriptionName = "uniphar-dev", CostFrom = DateTime.UtcNow.AddDays(-31), CostTo = DateTime.UtcNow.AddDays(-1) });
```

Comment:
should we (not sure at this time) check for the effective date, not all months have 31 days, also not sure if this runs on the last day of the month, or on the first..

or if we always want the last 31 days. I'm thinking that if we ask for the last 31 days, we might double charge people

maybe something like 

```csharp
DateTime now = DateTime.Now;
var startDate = new DateTime(now.Year, now.Month, 1);
var endDate = startDate.AddMonths(1).AddDays(-1);
```

or 

```csharp
var today = DateTime.Today;
var monthStart = new DateTime(today.Year, today.Month, 1);
var monthEnd = monthStart.AddMonths(1).AddDays(-1);
var lastMonthStart = monthStart.AddMonths(-1);
var lastMonthEnd = monthStart.AddDays(-1);
```

would work best? If this is just a mock, ignore...